blueman/main/DhcpClient.py

```diff
@@ -37,7 +37,6 @@ def __init__(self, interface, timeout=30):
             if path:
                 self._command = [path] + command[1:] + [self._interface]
                 break
-        self._client = None
```

Comment:
Pycharm does not like but flake8 does. But we do create an instance attribute outside of ``__init__``. If flake8 is happy then fine :weary: